### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by daritter**
Hi Graeme,

I took the liberty of using your presentation which was very helpful. However there were some typos here and there so I tried to fix them.

### CODE REVIEW

File: presentation-insights.ipynb [hunk 1]
```diff
@@ -3736,7 +3736,7 @@
    "source": [
     "## Subclasses and Inheritance\n",
     "\n",
-    "Python classes can also inherit from other classes, becoming subclasses - this allows objects which extend or specilaise the classes that they inherit from in the usual object oriented way"
+    "Python classes can also inherit from other classes, becoming subclasses - this allows objects which extend or specialize the classes that they inherit from in the usual object oriented way"
```
Comment (graeme-a-stewart) [inline presentation-insights.ipynb]: This one I don't like - 's' is the standard spelling in UK English, with 'z' used by our American cousins.

File: presentation-insights.ipynb [hunk 2]
```diff
@@ -5721,7 +5721,7 @@
    "source": [
     "## Seaborn\n",
     "\n",
-    "Here's another extremely useful python library, the [*seaborn*](http://seaborn.pydata.org/index.html) statistical visulaisation library\n",
+    "Here's another extremely useful python library, the [*seaborn*](http://seaborn.pydata.org/index.html) statistical visualization library\n",
```
Comment (graeme-a-stewart) [inline presentation-insights.ipynb]: ditto, keep this as is

File: presentation-insights.ipynb [hunk 3]
```diff
@@ -5861,7 +5861,7 @@
     "\n",
     "* [numpy](http://www.numpy.org/) - The high performance core of all serious Python numerics\n",
     "* [pandas](https://pandas.pydata.org/) - Python Data Analysis package for importing working with bulk data\n",
-    "* [scipy](https://www.scipy.org/) - Lots of common scientific routines, such as minimisation\n",
+    "* [scipy](https://www.scipy.org/) - Lots of common scientific routines, such as minimization\n",
```
Comment (graeme-a-stewart) [inline presentation-insights.ipynb]: ditto

File: presentation-insights.ipynb [hunk 4]
```diff
@@ -6358,7 +6358,7 @@
    "source": [
     "# Final Words\n",
     "\n",
-    "In case you didn't yet realise it, **Python is pretty amazing**\n",
+    "In case you didn't yet realize it, **Python is pretty amazing**\n",
```
Comment (graeme-a-stewart) [inline presentation-insights.ipynb]: ditto

**Review by graeme-a-stewart — CHANGES_REQUESTED** [review]
Great - thanks so much for this. It really helps to have a fresh pair of eyes look over these things.

The only changes I don't want are the ones that are just UK->US spelling variations, where I would like to stick with the UK version.

**Comment by daritter** [comment]
Yes, I was assuming there might be some UK/US discussion here :D. I changed all the ones you requested

**Pull request merged**